docs/release_process.md

```diff
@@ -69,51 +136,83 @@ controlled by the `.github/workflows/release-changelog.json` configuration file.
 
 The general rule is that the PR Titles will compose the body of the changelog.
 
+### Tagging
 
-- Create and push a **vX.Y.Z** tag to the GitHub repo. Wait for the release action
-to finish.\
-  `git tag vX.Y.Z stabilization-vX.Y.Z`\
-  `git push --tags`
-
+- Create and push a **vX.Y.Z** tag to the GitHub repo.
+```
+git tag vX.Y.Z stabilization-vX.Y.Z
+git push --tags
+```
+- Wait for the release action to finish.
 - Check the release draft and update the release notes if necessary.
 
 - Publish the release.
 
-In case there is a need to run the job again, delete the release
-draft and run the GitHb Action again.
+_NOTE: In case there is a need to run the job again, delete the release
+draft and run the GitHb Action again._
 
+# Clean Up
 
-Clean Up
--------------------------
-
-- Update the **stable** branch to point to the new release:\
- `git checkout stable`\
- `git merge stabilization-vX.Y.Z`
+- Update the **stable** branch to point to the new release:
+```
+git checkout stable
+git merge stabilization-vX.Y.Z
+```
```

Comment:
I'd add here that some conflicts may have to be solved.

Reply:
Yes, why not? : )